lcio/trackerrawdata.go

```diff
@@ -19,10 +19,11 @@ type TrackerRawDataContainer struct {
 }
 
 type TrackerRawData struct {
-	CellID0 int32
-	CellID1 int32
-	Time    int32
-	ADCs    []uint16
+	CellID0    int32
+	CellID1    int32
+	Time       int32
+	nADCValues int32
+	ADCValues  []uint16
```

Comment:
I am relatively agnostic wrt this but, is there a fundamental reason to rename this as `ADCValues` ?